src/AutorestSwift/View Models/KeyValueViewModel.swift

```diff
@@ -31,6 +31,7 @@ import Foundation
 ///     "key" = value
 struct KeyValueViewModel {
     let key: String
+    let paramName: String?
```

Comment:
Could you update the example in the doc string? It would make it clearer without having to look at the stencil file what it's role is in the code gen.

Reply:
Added doc string